.pipelines/templates/json.yml

```diff
@@ -0,0 +1,43 @@
+# This Yaml Document has been converted by ESAI Yaml Pipeline Conversion Tool.
+# Please make sure to check all the converted content, it is your team's responsibility to make sure that the pipeline is still valid and functions as expected.
+parameters:
+  parentJobs: []
+jobs:
+- job: json
+  variables:
+  - name: runCodesignValidationInjection
+    value: false
+  - name: NugetSecurityAnalysisWarningLevel
+    value: none
+  - name: ob_outputDirectory
+    value: '$(Build.ArtifactStagingDirectory)/ONEBRANCH_ARTIFACT'
+  - group: AzureBlobServiceConnection
+  displayName: Create Json for Blob
+  dependsOn: ${{ parameters.parentJobs }}
+  condition: succeeded()
+  pool:
+    type: windows
+  steps:
+  - checkout: self
+    clean: true
+  - template: /.pipelines/templates/SetVersionVariables.yml@self
+    parameters:
+      ReleaseTagVar: $(ReleaseTagVar)
+      CreateJson: yes
+  - task: AzureFileCopy@4
```

Comment:
We should consider separating the build into a build and release jobs

Reply:
This is not used in the pipeline